### ISSUE TITLE
removed d6 stuff?

### ISSUE THREAD

**Issue by Ph0rk0z** [aguaviva/mpow-dsd6#1]
You removed all the D6 stuff? What's the point then. I'm trying to get something out of this band without coding the whole thing anew.

**Comment by aguaviva** [comment]
yeah, my bands were dying little by little as teh battery was wearing out. So I decided to remove the D6 support and just use the cellphone step counter. But you can still use the previous version that had the D6 support though.
